### PR TITLE
Add the Pipeline and GermanPipeline classes

### PR DESCRIPTION
**Pull request by danieldk**

### CODE REVIEW

**Comment by danieldk** [comment]
This probably doesn't need as strict a review. The workbench is mostly for internal use (small interactive environment to inspect sticker output).

File: sticker_workbench/cleanup.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    '≦': "<=",
+    '≧': ">=",
+    '…': "..."}
+
+
+def cleanup(text):
+    cleaned = str()
+
+    for ch in unicodedata.normalize('NFC', text):
+        if ch in REPLACEMENTS:
```
Comment (twuebi) [inline sticker_workbench/cleanup.py]: Maybe something like this?
`cleaned += REPLACEMENTS.get(ch,ch)`

or:

```py
def cleanup(text):
    norm_it = unicodedata.normalize('NFC', text)
    return str(REPLACEMENTS.get(ch,ch) for ch in norm_it)
```

Comment (danieldk) [reply sticker_workbench/cleanup.py]: Good idea, fixed!

**Review by twuebi — APPROVED** [review]
Looks good

**Pull request merged**